topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.md

```diff
@@ -431,28 +431,33 @@ with predicted functional effects.
 
 > ### {% icon hands_on %} Hands-on: Variant Annotation
 >
-> To annotate our variants we are going to use **SnpEff** with the
-> `athalianaTair10` genome database as the source of functional genomic
+> To annotate our variants we are going to use **SnpEff** with its
+> `Arabidopsis_thaliana` genome database as the source of functional genomic
 > annotations, then generate a human-friendly variant report with the
 > **MiModD Report Variants** {% icon tool %} tool.
 >
-> 1. Use **SnpEff Download** {% icon tool %} (Galaxy tool version 4.1.0) to
->    download genome database `athalianaTair10`.
+> 1. Use **SnpEff Download** {% icon tool %} to download genome database
```

Comment:
Is it possible  to add the link to the tool there?

Reply:
I think we need to do this for the entire tutorial. Lets ship those enhancements first.